packages/asc-ui/src/internals/Box/BoxWrapper.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react'
+import styled from '../../styled-components'
+
+export type Props = {}
+
+export const BoxWrapperStyle = styled.div`
```

Comment:
Are internals only used by a specific UI? Or should they be globally available to other UIs in the future as well?

Reply:
just internal used